chart/templates/deployment.yaml

```diff
@@ -13,9 +13,9 @@ spec:
       {{- include "lava-gitlab-runner.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
       annotations:
         checksum/secret: {{ include (print $.Template.BasePath "/lava-token-secret.yaml") . | sha256sum }}
+      {{- with .Values.podAnnotations }}
```

Comment:
I hope @sjoerd will not mind the weird indentation here, but in that case we can fix it later, doing the patch this way makes it as clear as possible. :)